nifi-docs/src/main/asciidoc/walkthroughs.adoc

```diff
@@ -1024,59 +1018,80 @@ NOTE: The `nifi.cluster.load.balance.host=` entry must be manually populated her
+* Update the following lines:
+** In the `<userGroupProvider>` section, `<property name="Initial User Identity 1"></property>` -> `<property name="Initial User Identity 1">CN=my_username</property>` -- Adds an initial user with the DN generated in the client certificate. This is line 52 as of 1.11.4
+** In the `<userGroupProvider>` section, add the line `<property name="Initial User Identity 2">CN=node1.nifi, OU=NIFI</property>` -- Adds an initial user for `node1`
+** In the `<userGroupProvider>` section, add the line `<property name="Initial User Identity 3">CN=node2.nifi, OU=NIFI</property>` -- Adds an initial user for `node2`
+** In the `<userGroupProvider>` section, add the line `<property name="Initial User Identity 4">CN=node3.nifi, OU=NIFI</property>` -- Adds an initial user for `node3`
 +
-image::tls-authorizers-file-userGroupProvider.png[userGroupProvider section in the authorizers file]
-[start=5]
-. Locate the ``<accessPolicyProvider>`` section.
-. Locate the line ``<property name="Node Identity 1"></property>``
-. Add the CN and OU to the property: ``<property name="Node Identity 1">CN=nifi1.xyz.com, OU=NIFI</property>``
-. Add the following lines to specify the node identity for the remaining nodes:
-----
-<property name="Node Identity 2">CN=nifi2.xyz.com, OU=NIFI</property>
-<property name="Node Identity 3">CN=nifi3.xyz.com, OU=NIFI</property>
-----
+image::authorizers-xml-initial-user-identities.png["authorizers.xml with Initial User Identities"]
+** In the `<accessPolicyProvider>` section, `<property name="Initial Admin Identity"></property>` -> `<property name="Initial Admin Identity">CN=my_username</property>` -- Adds an initial admin with the DN generated in the client certificate. This is line 280 as of 1.11.4
+** In the `<accessPolicyProvider>` section, `<property name="Node Identity 1"></property>` -> `<property name="Node Identity 1">CN=node1.nifi, OU=NIFI</property>` -- Adds an initial node with the DN generated in the `node1` certificate. This is line 282 as of 1.11.4
+** In the `<accessPolicyProvider>` section, add the line `<property name="Node Identity 2">CN=node2.nifi, OU=NIFI</property>`
+** In the `<accessPolicyProvider>` section, add the line `<property name="Node Identity 3">CN=node3.nifi, OU=NIFI</property>`
 +
-image::tls-authorizers-file-accessPolicyProvider.png[accessPolicyProvider section in the authorizers file]
+image::authorizers-xml-initial-node-identities.png["authorizers.xml with Initial Node Identities"]
+* `cp node1.nifi/conf/authorizers.xml node2.nifi/conf/authorizers.xml` -- Copies the modified `authorizers.xml` file from `node1` to `node2`
+* `cp node1.nifi/conf/authorizers.xml node3.nifi/conf/authorizers.xml` -- Copies the modified `authorizers.xml` file from `node1` to `node3`
+. By default, NiFI waits for nodes to join for 5 minutes before the cluster is available. Because the number of nodes is known, this delay can be modified on each node to start up much faster. (Optional)
```

Comment:
Typo: "NiFI" should be "NiFi".
```suggestion
. By default, NiFi waits for nodes to join for 5 minutes before the cluster is available. Because the number of nodes is known, this delay can be modified on each node to start up much faster. (Optional)
```